tests/Controller/CRUDControllerTest.php

```diff
@@ -2476,19 +2451,8 @@ public function testCreateActionAjaxErrorWithoutAcceptApplicationJson(): void
             ->method('createView')
             ->willReturn($formView);
 
-        $this->expectTranslate('flash_create_error', [
-            '%name%' => '',
-        ], 'SonataAdminBundle');
-
-        $this->assertInstanceOf(Response::class, $response = $this->controller->createAction());
-        $this->assertSame($this->admin, $this->parameters['admin']);
```

Comment:
Why these expectations were removed?

Reply:
Becase [`handleXmlHttpRequestErrorResponse` used to return `null` when the `Accept` parameter was missing](https://github.com/sonata-project/SonataAdminBundle/blob/master/src/Controller/CRUDController.php#L1524) and now it return a Response, so [it doesn't execute some code](https://github.com/sonata-project/SonataAdminBundle/blob/master/src/Controller/CRUDController.php#L340).